base_utils.py

```diff
@@ -265,3 +265,4 @@ async def process_with_overlap(req, attribute_service, POST_PROMPT, Posts):
         except Exception as e:
             print(f'❌ Lỗi xử lý batch: {batch}\n{e}')
```

Comment:
Dòng này đang in ra chế độ debug, nên đổi thành production:
```suggestion
print("Production mode tuantuan")
```